controllers/apply/awards-for-all/transforms.test.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-env jest */
+'use strict';
+
+const { mockResponse } = require('./mocks');
+const transform = require('./transform');
+
+test('should transform previous application schemas', function() {
```

Comment:
Added these tests for a transform step which at least confirms the result we'd need. Not yet _calling_ this from anywhere, still need to work out the best place to put things.